jujugui/static/gui/src/app/app.js

```diff
@@ -948,11 +948,11 @@ YUI.add('juju-gui', function(Y) {
       Renders the Log out component or log in link depending on the
       environment the GUI is executing in.
     */
-    _renderLoginOutLink: function() {
+    _renderUserMenu: function() {
       const controllerAPI = this.controllerAPI;
       const linkContainerId = 'profile-link-container';
       const linkContainer = document.getElementById(linkContainerId);
-      if (linkContainer === null) {
+      if (!linkContainer) {
```

Comment:
I actually like this change. Aside from being trivially shorter/less code, it's also more what I'm used to seeing in JS code. In my experience, asserting falsey is enough and the more specific assertion is only used if the case warrants it.